providers/gce/gce_loadbalancer_internal.go

```diff
@@ -626,6 +659,10 @@ func (g *Cloud) ensureInternalInstanceGroup(name, zone string, nodes []*v1.Node)
 // ensureInternalInstanceGroups generates an unmanaged instance group for every zone
 // where a K8s node exists. It also ensures that each node belongs to an instance group
 func (g *Cloud) ensureInternalInstanceGroups(name string, nodes []*v1.Node) ([]string, error) {
+	// Filter out any node that is not from the default network. This is required for multi-subnet feature.
+	// This should not change behavior for nodes that are in the default network.
+	// This can't be done earlier when listing node since the code is shared between internal and external LBs.
+	nodes = filterOutNodesFromNonDefaultNetworks(nodes, g.SubnetworkURL())
```

Comment:
We need to handle SubnetworkURL parse failure, even if we think it's rare.

Reply:
done